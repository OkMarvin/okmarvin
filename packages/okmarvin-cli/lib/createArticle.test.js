'use strict'
const createArticle = require('./createArticle')
describe('createArticle', () => {
  it('should warn when no title', () => {
    expect(
      () => createArticle('post')
    ).toThrow()
  })
  it('should pass', () => {
    beforeAll(() => {
      process.env.NODE_ENV = 'development'
    })
    expect(
      () => createArticle('post', 'okmarvin is static site generator')
    ).not.toThrow()
  })
})
