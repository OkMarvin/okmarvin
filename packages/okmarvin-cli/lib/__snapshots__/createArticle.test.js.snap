// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createArticle should warn when no title 1`] = `"[31mPlease specify post title[39m"`;
